examples/linear_model/plot_lasso_coordinate_descent_path.py

```diff
@@ -19,30 +17,39 @@
 
 from sklearn.linear_model import lasso_path, enet_path
 from sklearn import datasets
+from scipy import sparse
+
+print(__doc__)
 
 diabetes = datasets.load_diabetes()
 X = diabetes.data
 y = diabetes.target
 
 X /= X.std(axis=0)  # Standardize data (easier to set the l1_ratio parameter)
 
+X = sparse.csr_matrix(X)
+
 # Compute paths
 
 eps = 5e-3  # the smaller it is the longer is the path
+tol = 1e-10
```

Comment:
How important is this to the example? I am trying to not make the example too complex.